src/main/kotlin/net/atos/zac/mail/MailjetClientHelper.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.mail
+
+import com.mailjet.client.ClientOptions
+import com.mailjet.client.MailjetClient
+
+object MailjetClientHelper {
+    @JvmStatic
+    fun createMailjetClient(mailjetApiKey: String?, mailjetSecretKey: String?): MailjetClient {
+        return MailjetClient(
+            ClientOptions.builder()
+                .apiKey(mailjetApiKey)
+                .apiSecretKey(mailjetSecretKey)
+                .build()
+        )
+    }
+}
```

Comment:
No need for those `@JvmStatic` annotations. In this case this can just be a pure function. Pretty sure those arguments should be non-nullable?
```suggestion
fun createMailjetClient(mailjetApiKey: String, mailjetSecretKey: String) = 
MailjetClient(
            ClientOptions.builder()
                .apiKey(mailjetApiKey)
                .apiSecretKey(mailjetSecretKey)
                .build()
        )
```

Reply:
Removed static. Now we use `BronnenKt` in Java classes